_topic_map.yml

```diff
@@ -1626,6 +1626,9 @@ Topics:
 # Release notes
 - Name: Release Notes
   File: serverless-release-notes
+# Support
+- Name: Getting support
```

Comment:
I'm not sure about the title. This article doesn't provide a manual how to get support for Serverless. Maybe just "Support" like it is in the Service Mesh docs? These docs will be related to providing information `for` support.